src/runtime/virtcontainers/hypervisor.go

```diff
@@ -112,6 +126,66 @@ var commonVirtioblkKernelRootParams = []Param{ //nolint: unused, deadcode, varch
 	{"rootfstype", "ext4"},
 }
 
+func NvdimmKernelRootParams(rootfstype string) []Param {
```

Comment:
Please add tests for these new NvdimmKernelRootParams/NvdimmNoDAXKernelRootParams/VirtioblkKernelRootParams  
 functions.

Reply:
I added unit tests to cover all rootfs type corresponding kernel root parameters.